src/Tizen.NUI/src/internal/EXaml/Action/GetObjectByPropertyAction.cs

```diff
@@ -61,7 +61,7 @@ public void Init()
 
         public void OnActive()
         {
-            if (null != childOp)
+            if (null != childOp && null != (childOp.ValueList[0] as Instance))
```

Comment:
Hello, how about change this as like this?

`if (null != childOp && childOp.ValueList[0] is Instance valueList0)`
and 
`int instanceIndex = valueList0.Index;`

Reply:
> Hello, how about change this as like this?
> 
> `if (null != childOp && childOp.ValueList[0] is Instance valueList0)`
> and
> `int instanceIndex = valueList0.Index;`

Fixed